packages/api-aco/src/folder/folder.so.ts

```diff
@@ -145,6 +146,22 @@ export const createFolderOperations = (
                 await cms.deleteEntry(model, id);
                 return true;
             });
+        },
+        getFolderHierarchyById({ id }) {
+            return withModel(async model => {
+                const folder = await getFolder({ id });
+
+                const [entries] = await cms.listLatestEntries(model, {
+                    where: {
+                        type: folder.type
+                    },
+                    limit: 10000
+                });
+
+                const folders = entries.map(entry => getFolderFieldValues(entry, baseFields));
```

Comment:
Instead of copy pasting this chunk here, can we use `this.listFolders()` instead?

And once you do that, you no longer need to have a method in storage operations. You can just have this logic in `folder.crud.ts`, and have less code overall.